docs/developer-guide/etendo-copilot/available-tools/openapi-tool.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "OpenAPI interaction with Copilot"
+tags:
+    - Copilot
+    - IA
+    - Machine Learning
+    - OpenAPI
+    - Swagger
+    - API
+---
+:octicons-package-16: Javapackage: com.etendoerp.copilot.openapi
+
+:octicons-package-16: Javapackage: com.etendoerp.copilot.openapi.purchase
+
+
+## Overview
+The OpenAPI Specification (OAS) defines a standard, language-agnostic interface to RESTful APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection. When properly defined, a consumer can understand and interact with the remote service with a minimal amount of implementation logic. Similar to what interfaces have done for lower-level programming, OAS removes the guesswork in calling the service. For more information, see the [OpenAPI Initiative](https://www.openapis.org/).
+
+This Specification can be very usefull for Copilot, because can be used as guide to Copilot to interact with the services, without the need to know the implementation details and will not be necessary to implement specific code for each service.
+
+
+## Tools
+
+**Etendo API Tool**: This tool returns the OpenAPI Specification of the Etendo API, can be used by Copilot to ask for the general information of the API (endpoints, and the descriptions of the endpoints) or the specific information of the endpoints(Parameters, responses, etc). At this moment, the Etendo API Tool only returns the OpenAPI Specification of the Etendo API for insert purchase orders in Etendo.
+
+**API Call Tool**: This tool allows to Copilot to make a call to a API endpoint. 
```

Comment:
More explanation of the operation, as this is the tool that should be most understood.